sdk/storage/azure-storage-blob-batch/README.md

```diff
@@ -0,0 +1,330 @@
+# Azure Storage Blobs Batch client library for Java
+
+Azure Blob storage is Microsoft's object storage solution for the cloud. Blob
+storage is optimized for storing massive amounts of unstructured data.
+Unstructured data is data that does not adhere to a particular data model or
+definition, such as text or binary data.
```

Comment:
Add another line here to describe the purpose of this module specifically. Something like "This client library provides APIs to perform bulk/batch operations on Azure Blob storage."